### PR TITLE
modified header to look like mock

### PR DESCRIPTION
**Pull request by EtoKruto**
'modified header to look like mock - used user, if defined, to show icons: inbox, notification and profile'

### CODE REVIEW

File: client/src/components/Header.tsx [hunk 1]
```diff
@@ -95,6 +96,17 @@ const useStyles = makeStyles()((theme: Theme) => ({
     marginLeft: '10px',
     width: '100px',
   },
+  menuButton: {
+    color: '#323232',
+    fontFamily: 'Poppins',
```
Comment (esteban-gs) [inline client/src/components/Header.tsx:101]: Font family is set at the theme level, so, you could remove it.
```suggestion
```

File: client/src/components/Header.tsx [hunk 2]
```diff
@@ -95,6 +96,17 @@ const useStyles = makeStyles()((theme: Theme) => ({
...
+    fontSize: '1rem',
+    fontStyle: 'normal',
+    fontWeight: 400,
+    lineHeight: 'normal',
+    letterSpacing: '0.00938rem',
+    width: '100%',
+    margin: '0 5px 0 5px',
```
Comment (esteban-gs) [inline client/src/components/Header.tsx:108]: Please check if any of these aren't already set at the theme level.

File: client/src/components/Header.tsx [hunk 3]
```diff
@@ -171,164 +183,189 @@ function Header() {
...
+                id="exchange-button"
+                aria-label="exchange dropdown"
+                aria-controls={isExchangeMenuOpen ? 'about-menu' : undefined}
+                aria-haspopup="true"
+                aria-expanded={isExchangeMenuOpen ? 'true' : undefined}
+                onClick={handleClick}
+                endIcon={isExchangeMenuOpen ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+                sx={{
+                  textTransform: 'capitalize',
+                  color: '#323232',
```
Comment (esteban-gs) [inline client/src/components/Header.tsx:201]: It'll be more maintainable to get the primary/secondary color from the theme definition.

`theme.palette.primary.main`

Theme is imported from `src/theme/index.tsx`

**Review by esteban-gs — APPROVED** [review]
@alexislcstone , Thanks for working on this! I added a couple of comments for improvements.

I think we could also benefit from splitting up `Header` into smaller components, but up to you if want to tackle it in this PR.

**Comment by EtoKruto** [comment]
> @alexislcstone , Thanks for working on this! I added a couple of comments for improvements.
> 
> I think we could also benefit from splitting up `Header` into smaller components, but up to you if want to tackle it in this...

Totally think it is a good idea but rather leave it for another sprint or for someone else to split, there is some refactoring too I can see. @esteban-gs I addressed the questions and will be pushing to main here shortly.

**Pull request merged**